tests/api_test/api_test.cpp

```diff
@@ -886,6 +887,236 @@ bind_tailcall_test(_In_ struct bpf_object* object)
     WSACleanup();
 }
 
+#define SOCKET_TEST_PORT 0x3bbf
+
+void
+connect_send_udp_ipv4_traffic()
```

Comment:
nit: `connect_send_udp_ipv4_traffic` and `connect_send_udp_ipv6_traffic` seem to contain lot of duplicate code -- can we merge them to a single function that takes address family as input?